odoo/netsvc.py

```diff
@@ -16,6 +16,11 @@
 from . import sql_db
 from . import tools
 
+try:
+    from pythonjsonlogger import jsonlogger
+except:
```

Comment:
Never do bare except clauses, bad practice... try `except ImportError`

Furthermore, why the extra dependency? Can't we import the standard `json` lib and then make the `JSONFormatter` class do all the work?

Reply:
Ok, thank you for helping me improve my code style! It actually is a full blown `pep8` violation... I'll correct it.

I was thinking about just copying the whole file from upstream, it's one single file of about 20 LOC... But thing is I personally trust more in the experience of intricate use cases covered by a battle tested upstream library..

Another nice side effect of using a library: We don't need to introduce a configuration flag, so it will be used only by most advanced users by simply providing the library. What's you opinion on that?

Reply:
https://github.com/madzak/python-json-logger/blob/master/src/pythonjsonlogger/jsonlogger.py

Reply:
Thinking about a config option would as well be interesting, please let me know what you prefer... I have no problem in refactoring...